test/lib/jdk/test/lib/process/ProcessTools.java

```diff
@@ -399,7 +399,7 @@ private static List<String> addMainWrapperArgs(String mainWrapper, List<String>
 
         final List<String> unsupportedArgs = List.of(
              "-jar", "-cp", "-classpath", "--class-path", "--describe-module", "-d",
-             "--dry-run", "--list-modules","--validate-modules", "-version");
+             "--dry-run", "--list-modules","--validate-modules", "-m", "--module", "-version");
```

Comment:
Can you fix the indentation on line 398 above?

Reply:
It doesn't seem to make much sense to have an argument be in both `unsupportedArgs` and `doubleWordArgs`. `unsupportedArgs` seems to take precedence and the arg will never have a change to be found in `doubleWordArgs`.

Reply:
fixed